pkg/services/ngalert/api/tooling/definitions/provisioning_alert_rules.go

```diff
@@ -191,20 +191,19 @@ type AlertRuleGroup struct {
 }
 
 func (a *AlertRuleGroup) ToModel() (models.AlertRuleGroup, error) {
-	rules := make([]models.AlertRule, 0, len(a.Rules))
+	ruleGroup := models.AlertRuleGroup{
+		Title:     a.Title,
+		FolderUID: a.FolderUID,
+		Interval:  a.Interval,
+	}
 	for i := range a.Rules {
 		converted, err := a.Rules[i].UpstreamModel()
 		if err != nil {
 			return models.AlertRuleGroup{}, err
 		}
-		rules = append(rules, converted)
+		ruleGroup.Rules = append(ruleGroup.Rules, converted)
```

Comment:
Nice, this code is much cleaner than how I would have done it, great use of language features :smile: